dune/gdt/tools/hyperbolic.hh

```diff
@@ -52,7 +52,7 @@ double estimate_dt_for_hyperbolic_system(
   auto local_state = state.local_function();
   for (auto&& element : elements(grid_view)) {
     local_state->bind(element);
-    for (const auto& quadrature_point : QuadratureRules<D, d>::rule(element.type(), local_state->order())) {
+    for (auto&& quadrature_point : QuadratureRules<D, d>::rule(element.type(), local_state->order())) {
```

Comment:
you've had `[point, weight] = ...` above. Don't we have to change into that here as well?

Reply:
I've tried both to squash that dangling ref warning. We should settle on one form though yeah.